saleor/plugins/avatax/plugin.py

```diff
@@ -421,6 +423,63 @@ def get_tax_rate_type_choices(self, previous_value: Any) -> List[TaxType]:
             for tax_code, desc in get_cached_tax_codes_or_fetch(self.config).items()
         ]
 
+    def get_checkout_tax_rate(
+        self,
+        checkout: "Checkout",
+        product: "Product",
+        address: Optional["Address"],
+        checkout_line: "CheckoutLine",
+        discounts: Iterable[DiscountInfo],
+        previous_value: Decimal,
+    ) -> Decimal:
```

Comment:
Maybe we could pass `CheckoutLineInfo` instead of `product` and `checkout_line`

Reply:
I was wondering why we need particular checkout line until I noticed this method is getting tax for the actual line instead of checkout. Renaming would help a lot as already pointed out.